### ISSUE TITLE
After Installing Tensorflow when I TRy to verify I getting Follwing error can some body help on this

### ISSUE THREAD

**Issue by AvinashTiwari** [tensorflow/tensorflow#23506]
<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:bug_template</em>

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow):
    NO custom code or Module
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
    Windows 8.1 (64 bit)
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
   No mobile device
- TensorFlow installed from (source or binary):
   binary 
- TensorFlow version (use command below):
    >pip install tensorflow
    >pip3 install tensorflow
    Currently default comes as tensorflow
C:\Phyton>pip3 show tensorflow
Name: tensorflow
Version: 1.11.0
Summary: TensorFlow is an open source machine learning framework for everyone.
Home-page: https://www.tensorflow.org/
Author: Google Inc.
Author-email: opensource@google.com
License: Apache 2.0
Location: c:\users\avinash.t\anaconda3\lib\site-packages
Requires: termcolor, astor, tensorboard, keras-applications, numpy, grpcio, whee
l, keras-preprocessing, setuptools, six, protobuf, gast, absl-py
Required-by:
 
- Python version:
 3.6.7
- Bazel version (if compiling from source):
No compilation used 
- GCC/Compiler version (if compiling from source):
No compilation used 
- CUDA/cuDNN version:
No CUDA
- GPU model and memory:
No GPU

You can collect some of this information using our environment capture [script](https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh)
You can also obtain the TensorFlow version with
python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"
Version is :- 1.11
When i execute the above command following error occurs

Failed to load the native TensorFlow runtime.

**Describe the current behavior**
   LOG is Posted
**Describe the expected behavior**

**Code to reproduce the issue**
Provide a reproducible test case that is the bare minimum necessary to generate the problem.

**Other info / logs**
C:\Users\avinash.t\Anaconda3>python
Python 3.6.7 |Anaconda, Inc.| (default, Oct 28 2018, 19:44:12) [MSC v.1915 64 bi
t (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import tensor
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'tensor'
>>> import tensorflow as tf
Traceback (most recent call last):
  File "C:\Users\avinash.t\Anaconda3\lib\site-packages\tensorflow\python\pywrap_
tensorflow.py", line 58, in <module>
    from tensorflow.python.pywrap_tensorflow_internal import *
  File "C:\Users\avinash.t\Anaconda3\lib\site-packages\tensorflow\python\pywrap_
tensorflow_internal.py", line 28, in <module>
    _pywrap_tensorflow_internal = swig_import_helper()
  File "C:\Users\avinash.t\Anaconda3\lib\site-packages\tensorflow\python\pywrap_
tensorflow_internal.py", line 24, in swig_import_helper
    _mod = imp.load_module('_pywrap_tensorflow_internal', fp, pathname, descript
ion)
  File "C:\Users\avinash.t\Anaconda3\lib\imp.py", line 243, in load_module
    return load_dynamic(name, filename, file)
  File "C:\Users\avinash.t\Anaconda3\lib\imp.py", line 343, in load_dynamic
    return _load(spec)
ImportError: DLL load failed: A dynamic link library (DLL) initialization routin
e failed.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\avinash.t\Anaconda3\lib\site-packages\tensorflow\__init__.py",
line 22, in <module>
    from tensorflow.python import pywrap_tensorflow  # pylint: disable=unused-im
port
  File "C:\Users\avinash.t\Anaconda3\lib\site-packages\tensorflow\python\__init_
_.py", line 49, in <module>
    from tensorflow.python import pywrap_tensorflow
  File "C:\Users\avinash.t\Anaconda3\lib\site-packages\tensorflow\python\pywrap_
tensorflow.py", line 74, in <module>
    raise ImportError(msg)
ImportError: Traceback (most recent call last):
  File "C:\Users\avinash.t\Anaconda3\lib\site-packages\tensorflow\python\pywrap_
tensorflow.py", line 58, in <module>
    from tensorflow.python.pywrap_tensorflow_internal import *
  File "C:\Users\avinash.t\Anaconda3\lib\site-packages\tensorflow\python\pywrap_
tensorflow_internal.py", line 28, in <module>
    _pywrap_tensorflow_internal = swig_import_helper()
  File "C:\Users\avinash.t\Anaconda3\lib\site-packages\tensorflow\python\pywrap_
tensorflow_internal.py", line 24, in swig_import_helper
    _mod = imp.load_module('_pywrap_tensorflow_internal', fp, pathname, descript
ion)
  File "C:\Users\avinash.t\Anaconda3\lib\imp.py", line 243, in load_module
    return load_dynamic(name, filename, file)
  File "C:\Users\avinash.t\Anaconda3\lib\imp.py", line 343, in load_dynamic
    return _load(spec)
ImportError: DLL load failed: A dynamic link library (DLL) initialization routin
e failed.

Failed to load the native TensorFlow runtime.

See https://www.tensorflow.org/install/install_sources#common_installation_probl
ems

for some common reasons and solutions.  Include the entire stack trace
above this error message when asking for help.
>>>

**Comment by Kalevera** [comment]
I'm getting the same error after migrating to Tensorflow 1.11 e cwpt I'm pretty sure my error is related to cuDNN64_90.dll not being found (I'm on cuFDNN64_91.dll) 

You might try down grading Tensorflow back to 1.5 or 1.7

**Comment by AvinashTiwari** [comment]
C:\Users\avinash.t>python
Python 3.6.7 |Anaconda, Inc.| (default, Oct 28 2018, 19:44:12) [MSC v.1915 64 b
t (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import tensorflow as tf
Traceback (most recent call last):
  File "C:\Users\avinash.t\Anaconda3\lib\site-packages\tensorflow\python\pywrap
tensorflow_internal.py", line 14, in swig_import_helper
    return importlib.import_module(mname)
  File "C:\Users\avinash.t\Anaconda3\lib\importlib\__init__.py", line 126, in i
port_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 658, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 571, in module_from_spec
  File "<frozen importlib._bootstrap_external>", line 922, in create_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
ImportError: DLL load failed: A dynamic link library (DLL) initialization routi
e failed.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\avinash.t\Anaconda3\lib\site-packages\tensorflow\python\pywrap
tensorflow.py", line 58, in <module>
    from tensorflow.python.pywrap_tensorflow_internal import *
  File "C:\Users\avinash.t\Anaconda3\lib\site-packages\tensorflow\python\pywrap
tensorflow_internal.py", line 17, in <module>
    _pywrap_tensorflow_internal = swig_import_helper()
  File "C:\Users\avinash.t\Anaconda3\lib\site-packages\tensorflow\python\pywrap
tensorflow_internal.py", line 16, in swig_import_helper
    return importlib.import_module('_pywrap_tensorflow_internal')
  File "C:\Users\avinash.t\Anaconda3\lib\importlib\__init__.py", line 126, in i
port_module
    return _bootstrap._gcd_import(name[level:], package, level)
ModuleNotFoundError: No module named '_pywrap_tensorflow_internal'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\avinash.t\Anaconda3\lib\site-packages\tensorflow\__init__.py",
line 24, in <module>
    from tensorflow.python import *  # pylint: disable=redefined-builtin
  File "C:\Users\avinash.t\Anaconda3\lib\site-packages\tensorflow\python\__init
_.py", line 49, in <module>
    from tensorflow.python import pywrap_tensorflow
  File "C:\Users\avinash.t\Anaconda3\lib\site-packages\tensorflow\python\pywrap
tensorflow.py", line 74, in <module>
    raise ImportError(msg)
ImportError: Traceback (most recent call last):
  File "C:\Users\avinash.t\Anaconda3\lib\site-packages\tensorflow\python\pywrap
tensorflow_internal.py", line 14, in swig_import_helper
    return importlib.import_module(mname)
  File "C:\Users\avinash.t\Anaconda3\lib\importlib\__init__.py", line 126, in i
port_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 658, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 571, in module_from_spec
  File "<frozen importlib._bootstrap_external>", line 922, in create_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
ImportError: DLL load failed: A dynamic link library (DLL) initialization routi
e failed.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\avinash.t\Anaconda3\lib\site-packages\tensorflow\python\pywrap
tensorflow.py", line 58, in <module>
    from tensorflow.python.pywrap_tensorflow_internal import *
  File "C:\Users\avinash.t\Anaconda3\lib\site-packages\tensorflow\python\pywrap
tensorflow_internal.py", line 17, in <module>
    _pywrap_tensorflow_internal = swig_import_helper()
  File "C:\Users\avinash.t\Anaconda3\lib\site-packages\tensorflow\python\pywrap
tensorflow_internal.py", line 16, in swig_import_helper
    return importlib.import_module('_pywrap_tensorflow_internal')
  File "C:\Users\avinash.t\Anaconda3\lib\importlib\__init__.py", line 126, in i
port_module
    return _bootstrap._gcd_import(name[level:], package, level)
ModuleNotFoundError: No module named '_pywrap_tensorflow_internal'


Failed to load the native TensorFlow runtime.

See https://www.tensorflow.org/install/install_sources#common_installation_prob
ems

for some common reasons and solutions.  Include the entire stack trace
above this error message when asking for help.
>>>
This is for tensorflow 1.7.0

**Comment by AvinashTiwari** [comment]
It worked with tensoflow == 1.5
C:\Users\avinash.t>python
Python 3.6.7 |Anaconda, Inc.| (default, Oct 28 2018, 19:44:12) [MSC v.1915 64 bi
t (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import tensorflow as tf
>>> print(tf.reduce_sum(tf.random_normal([1000, 1000])))
Tensor("Sum:0", shape=(), dtype=float32)
>>>

**Comment by Kalevera** [comment]
@mrry what dll's is TF1.5 looking for and TF1.7? 

Also built-in self check functions not returning correct dll codes or files that are missing when using TF1.11

**Comment by tensorflowbutler** [comment]
Nagging Assignee @mrry: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by mrry** [comment]
The self-check functions are once again available in the latest versions of TensorFlow (1.15 / 2.0), so please try again with a more recent version, and open a new issue if they don't work.
